XU-TechnicalPaper.md

```diff
@@ -25,163 +25,176 @@
 
-Each participating exchange runs its own XUD instance, which opens payment channels on each chain it chooses to support. This means, if an exchange wants to enable its users to trade bitcoin and litecoin through Exchange Union, it has to open a payment channel on the litecoin blockchain AND on the bitcoin blockchain to at least one other exchange in the union. Payment channels in XU are meant to be kept open long-term. Also, the current design doesn't encourage high volume transactions in one single channel and most implementations currently set upper limits as a 'training wheels restriction' (e.g. lnd's max. channel size is: 0.16777216 BTC). To achieve scalability, we will rely on multiple channels, which carry out partial transfers of one atomic payment as proposed in the [new AMP protocol by Laolu and Conner Fromknecht](https://lists.linuxfoundation.org/pipermail/lightning-dev/2018-February/000993.html). This allows:
-- to elegantly get around channel volume restrictions
-- to elegantly increase/decrease volume when needed by simply opening/closing new channels
-- achieve better connectivity (direct channels) between different XUDs
+With this mechanism, payment channels are trustless. Trustless means one bitcoin in a payment channel is backed by one locked bitcoin on the blockchain and can be transferred between two parties securely, even though only the opening and closing transaction are included in the blockchain. Simplified, this is secure, because all transactions are valid on the underlying blockchain and can be broadcast by either party at any time. So both parties can rest assured to be in possession of the respective funds when receiving a transaction from the other party in a channel.
 
+Payment channels also allow secure and trustless *routing* of funds through intermediaries using a technology called [Hashed Timelock Contracts (HTLCs)](https://en.bitcoin.it/wiki/Hashed_Timelock_Contracts), which eliminate the need for a direct channel between everyone. The [Lightning White Paper](https://lightning.network/lightning-network-paper.pdf) describes this in detail. This is useful since usually a few payment channels to some other nodes are enough to reach the rest of the network. The payment channel layer of XUD is comprised by the feature-rich [LND](https://github.com/lightningnetwork/lnd/) implementation for payment channels on BTC & LTC and by the [raiden](https://github.com/raiden-network/raiden/) implementation for payment channels on ETH, including all ERC20 tokens. XUD will rely on improved [auto-pilot](https://github.com/lightningnetwork/lnd/tree/master/autopilot) features to manage channels and routing fees automatically. Exact amounts of routing fees are hard to quote in advance of a trade and since we expect these to be negligibly low in future, exchanges are meant to use their [revenue from Exchange Union](#39-incentivisation---the-role-of-xuc) to offset routing fees.
```

Comment:
Maybe link to the lightning white paper up where Dryja & Poon are mentioned the first time.

Reply:
Done. It's linked twice now, but that should be fine.